docs/comparisons/testing/runtime.md

```diff
@@ -0,0 +1,68 @@
++++
+title = "Runtime"
++++
+
+# Runtime
+\toc
+
+## Introduction
+This page compares packages that test the runtime performance of code. This is also commonly know as "benchmarking". Julia base includes several important functions and macros that help you measure the runtime of your code, which are listed in the [reference for Julia base](https://docs.julialang.org/en/v1/base/base/). However, there are many clever tricks involved in good benchmarking, which are often best left to dedicated packages. If you want a gentle introduction to why we need such tricks, read [Why use packages to measure runtime](#why-use-packages-to-measure-runtime).
+
+## Overview
+The most popular and established packages is [BenchmarkTools.jl](#benchmarktools). The general recommendation is to use BenchmarkTools.
+
+{{star_history BenchmarkTools Chairmarks}}
+## Packages
+### BenchmarkTools
+{{badge BenchmarkTools}}
+BenchmarkTools is is widely used, and beloved by the Julia community, who cares a lot about fast code. It is easy to use, capable, and very much battle-tested.
+
+### Chairmarks
+{{badge Chairmarks}}
+Chairmarks.jl is a relatively new benchmarking package, which makes the following claim in it's [documentation](https://chairmarks.lilithhafner.com/v1.2.0/): "Chairmarks measures performance hundreds of times faster than BenchmarkTools without compromising on accuracy". So that is the upside. The downside is that it is not yet battle-tested. If want to run your benchmarks faster, then Chairmarks might be very interesting. This is particularly relevant for [packages that measure the runtime of other packages](#packages-that-measure-runtime-of-packages).
```

Comment:
```suggestion
Chairmarks.jl is a relatively new benchmarking package, which makes the following claim in it's [documentation](https://chairmarks.lilithhafner.com/stable/): "Chairmarks measures performance hundreds of times faster than BenchmarkTools without compromising on accuracy". So that is the upside. The downside is that it is not yet battle-tested. If want to run your benchmarks faster, then Chairmarks might be very interesting. This is particularly relevant for [packages that measure the runtime of other packages](#packages-that-measure-runtime-of-packages).
```